docs/userstories/v1/egress-control-in-anp/egress-control-cluster-nodes.md

```diff
@@ -0,0 +1,39 @@
+
+### Summary
+
+1. **As a** cluster administrator **I want** to easily block access from
```

Comment:
From my experience, there are more users asking for strong allow rules between workloads and k8s apiserver (when there are other deny-all rules applied to these workloads) than users who look to block workload <-> k8s apiserver communication. Maybe we can make the usecase a bit more generic here?

Reply:
ack, I have seen both asks but yes let me make this more generic to include both scenarios

Reply:
fixed, added a new use case to reflect the allow rules part as well.